server/db/verifyConnection.js

```diff
@@ -0,0 +1,21 @@
+const { Client } = require("pg");
+const { dbServerQuery } = require("./db-query");
+
+// send an initial query to the db server and make sure that it does not throw an error
+module.exports = {
+  async verifyConnection({ user, host, password, database, port }) {
```

Comment:
Is this file needed if there's already the same function in the `event-db-query.js` file? I don't see it being used anywhere so far

Reply:
Ah, good catch! I forgot to delete it while I was experimenting with different options.